docs/_includes/ds6/dialog.html

```diff
@@ -0,0 +1,455 @@
+<div id="dialog">
```

Comment:
Can you please use indentation of 4 spaces to match other files.

Reply:
👍